pkg/tsdb/cloudwatch/cloudwatch_test.go

```diff
@@ -111,7 +113,11 @@ func Test_CheckHealth(t *testing.T) {
 	t.Run("successfully query metrics and logs", func(t *testing.T) {
 		client = fakeCheckHealthClient{}
 		im := datasource.NewInstanceManager(func(s backend.DataSourceInstanceSettings) (instancemgmt.Instance, error) {
```

Comment:
Not a big deal, wondering here if we shouldn't just declare `im` once near the top of `Test_CheckHealth` since it has become a bit big. Can improve the focus and readability of the check health tests.